packages/design-system/src/components/HeaderBar/docs/HeaderBar.js

```diff
@@ -57,15 +56,15 @@ export const Portal = () => {
 			<HeaderBar.Content>
 				<HeaderBar.ContentRight>
 					<HeaderBar.Item freeze>
-						<Button.Primary small theme={dark}>
+						<ButtonPrimary onClick={() => {}} size="S">
```

Comment:
```suggestion
						<ButtonPrimary onClick={() => {}} size="<constant from the component>">
```

Reply:
In a TS library, that's not something I want to promote!